### ISSUE TITLE
Advice for field mapping with slice thickness < 2 mm

### ISSUE THREAD

**Issue by BenInglis** [CMRR-C2P/MB#88]
The Human Connectome Project used 2 mm slices in its fMRI protocols and was thus able to use the standard Siemens "gre_field_mapping" routine to acquire field maps. The gre_field_mapping sequence appears to be limited to 2 mm or thicker slices, necessitating a different approach to field mapping when pushing MB-EPI to < 2 mm slices. We are testing a (1.54 mm)^3 MB-EPI protocol for fMRI.

For field mapping I am considering:
1. Using cmrr_mbep2d_se with two different TEs
2. Using cmrr_mbep2d_se  with AP then PA phase encoding
3. Using cmrr_mbep2d_bold with AP then PA phase encoding

Now, in one article, "The Minimal Preprocessing Pipelines for the Human Connectome Project," 

http://europepmc.org/articles/pmc3720813

they discuss using SE with reversed phase encoding:

"To more rapidly measure the b0 field for correction EPI distortions, we acquire two spin echo EPI images with reversed phase encoding directions (60 seconds total for 3 pairs of images). (Note that we refer to these images as spin echo fieldmaps, though they measure the field by reversing the phase encoding direction, which is a very different mechanism from standard field maps that use a phase difference calculated from two different TEs). These spin echo EPI images have the same geometrical, echo spacing (0.58ms in our scans), and phase encoding direction parameters as the gradient echo fMRI scans. These images enable accurate correction for spatial distortions in the fMRI images so that these images can be precisely aligned with the structural images. Two of these spin echo EPI fieldmapping pairs are acquired in each functional session, for added robustness with respect to acquisition errors and subject movement, along with one set of B1− receive field-mapping images (with identical parameters to those described in the structural session)."

I am curious to know what other people are doing for field mapping of high resolution MB-EPI data. Any advice and tips greatly appreciated.

**Comment by rwmair** [comment]
Hi Ben,

Not sure if this is still an issue for you, as its an old post. However, the simplest way around this is to modify the Siemens gre_field_mapping sequence to permit slices thinner than 2mm.  However, if you're doing whole-brain, isotropic coverage, the scan time will balloon out to 3-4 mins for the field map. So using the cmrr_mbep2d_se with AP then PA phase encoding will be much more efficient (although requires a different post-processing routine)

**Comment by BenInglis** [comment]
Thanks Ross. Funnily enough, your note came in while we were sitting at the scanner testing exactly that: a modified version of gre_field_mapping. So far we have persuaded the POET sim to allow 1.5 mm slices but the compiled version doesn't, yet. Not sure why the discrepancy because we carefully matched every parameter then went through every possible control option (read b/w, matrix, lowest TE, phase encode FOV percent, etc. etc.) to try to persuade the compiled version below 2 mm, but it wouldn't oblige. Looks like the slice select refocus grad (with flow comp on or off) is at ceiling. But we haven't yet increased the duration of the excite RF, which is what we had to do a few years ago when we wanted a similarly thinner minimum slice for ep2d_bold. So modifying the RF duration is next. Once we get this working I'll post to this issue a summary of what we changed, in case anyone else wants to duplicate what we did.

We do have some people using AP then PA phase encoding, as used in the Human Connectome Project, but there is a concern over poor support in regions with high dropout for 1.5 mm isotropic voxels, TE~40 ms and 6/8 partial Fourier. We'd like the option to try both, even though as you say the scan time for the field map is going to be a few mins. (Given the long recon time for MB=6 on a Step IV imager perhaps that's fine :-)
